### ISSUE TITLE
Problem reading chapters 

### ISSUE THREAD

**Issue by JuanCRMorenoo** [Jays2Kings/tachiyomiJ2K#1538]
### Steps to reproduce

When I open the manga chapter, it won't let me read it, it closes automatically. It's hard for me to add a screenshot because it closes instantly; it happened from one day to the next. 

### Expected behavior

Being able to read the chapters without them closing 

### Actual behavior

Being able to read the chapters without them closing 

### Crash logs

App version: 1.6.4 (standard, 2080add1b, 102, 2023-03-16T16:59Z)
Android version: 12 (SDK 31)
Android build ID: S2SES32.28-70-1
Device brand: motorola
Device manufacturer: motorola
Device name: hawao
Device model: moto g42
Device product name: hawao_g

--------- beginning of main
04-06 13:55:03.451 11778 11821 E FrameEvents: updateAcquireFence: Did not find frame.
04-06 13:55:10.707 11778 11821 E FrameEvents: updateAcquireFence: Did not find frame.
04-06 13:55:17.042 11778 11821 E FrameEvents: updateAcquireFence: Did not find frame.
04-06 13:55:18.790 11778 11821 E FrameEvents: updateAcquireFence: Did not find frame.

### TachiyomiJ2K version

1.6.4

### Android version

Android 12

### Device

Moto g42

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I am reporting an issue exclusive to this fork. I have also checked that is not an issue on the [main version of Tachiyomi](https://github.com/tachiyomiorg/tachiyomi)
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[1.6.4](https://github.com/Jays2Kings/tachiyomiJ2K/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] I have filled out all of the requested information in this form.

**Comment by Jays2Kings** [comment]
please attach a video then, also add more details like source, manga, chapter

**Comment by BitSchism** [comment]
I'm experiencing the same bug (along with potentially others) that eventually lead to a crash. This can occur on any source, manga, chapter, etc. It can even occur on the home screen (but usually only after the app has been used to read something first).

Crash behavior varies from crashing from the chapter you are reading to the home screen to a full app crash to phone dashboard.

Crash Log below:

App version: 1.6.6 (standard, 21a2705c7, 104, 2023-04-19T22:38Z)
Android version: 13 (SDK 33)
Android build ID: TP1A.220624.014.S908USQS2CWE1
Device brand: samsung
Device manufacturer: samsung
Device name: b0q
Device model: SM-S908U
Device product name: b0qsqw

--------- beginning of main
05-28 13:26:13.992   453   678 E FrameEvents: updateAcquireFence: Did not find frame.
05-28 13:29:08.961   453   678 E FrameEvents: updateAcquireFence: Did not find frame.
05-28 13:29:17.653   453   678 E FrameEvents: updateAcquireFence: Did not find frame.
05-28 13:29:43.236   453   678 E FrameEvents: updateAcquireFence: Did not find frame.
05-28 13:29:45.675   453   678 E FrameEvents: updateAcquireFence: Did not find frame.
05-28 13:35:11.267   453   678 E FrameEvents: updateAcquireFence: Did not find frame.
05-28 14:47:10.166  1335  1335 E e.tachiyomi.j2k: Not starting debugger since process cannot load the jdwp agent.
05-28 14:47:10.733  1335  1335 E SQLiteLog: (283) recovered 762 frames from WAL file /data/data/eu.kanade.tachiyomi.j2k/databases/tachiyomi.db-wal
05-28 14:47:10.763  1335  1728 E SQLiteLog: (284) automatic index on C(manga_id)
05-28 14:47:10.763  1335  1728 E SQLiteLog: (284) automatic index on R(manga_id)
05-28 14:47:10.763  1335  1728 E SQLiteLog: (284) automatic index on B(manga_id)
05-28 14:47:10.763  1335  1728 E SQLiteLog: (284) automatic index on mangas_categories(manga_id)
05-28 14:47:10.803  1335  1335 E BufferQueueProducer: Unable to open libpenguin.so: dlopen failed: library "libpenguin.so" not found.
05-28 14:47:10.963  1335  1719 E SQLiteLog: (284) automatic index on C(manga_id)
05-28 14:47:10.964  1335  1719 E SQLiteLog: (284) automatic index on mangas(_id)



I've turned on automatic reporting, so I hope further logs will help. Thanks for all of your efforts!

**Comment by albyp501** [comment]
I also have the same problem, been having this problem ever since I switched from Tachiyomi to TachiyomiJ2K. Same problems with the ones above. I've noticed it happens whenever you scroll to the next chapter, it freezes. Theres also where if you leave the app to use another app, crashes due to loading to another chapter or page happens often.

App version: 1.7.3 (standard, b0a3e863aa, 110, 2023-10-26T04:09Z) 
Android version: 13 (SDK 33) 
Android build ID: TP1A.220624.014.A715FXXUADWH4 
Device brand: samsung 
Device manufacturer: samsung 
Device name: a71 
Device model: SM-A715F 
Device product name: a71nsxx 

Extensions that are outdated, obsolete, or unofficial 
Extension Name: ReadComicOnline 
Installed Version: 1.4.17 
Available Version: 1.4.18 
Obsolete: false Unofficial: false 

Extension Name: HBrowse
Installed Version: 1.4.7 
Available Version: N/A 
Obsolete: true 
Unofficial: false 

Extension Name: MangaPark v3 
Installed Version: 1.4.18 
Available Version: 1.4.19 
Obsolete: false 
Unofficial: false 

Extension Name: Aqua Manga 
Installed Version: 1.4.39 
Available Version: 1.4.40 
Obsolete: false 
Unofficial: false

 --------- beginning of main 
12-30 12:52:13.966 31017 31017 E e.tachiyomi.j2k: Not starting debugger since process cannot load the jdwp agent. 
12-30 12:52:14.194 31017 31106 E e.tachiyomi.j2k: No package ID 7a found for ID 0x7a0b0007. 
12-30 12:52:14.194 31017 31106 E e.tachiyomi.j2k: No package ID 7a found for ID 0x7a0b0007. 
12-30 12:52:15.643 31017 31017 E SQLiteLog: (283) recovered 55 frames from WAL file /data/data/eu.kanade.tachiyomi.j2k/databases/tachiyomi.db-wal 
12-30 12:52:15.702 31017 31333 E SQLiteLog: (284) automatic index on C(manga_id) 
12-30 12:52:15.702 31017 31333 E SQLiteLog: (284) automatic index on R(manga_id) 
12-30 12:52:15.702 31017 31333 E SQLiteLog: (284) automatic index on B(manga_id) 
12-30 12:52:15.702 31017 31333 E SQLiteLog: (284) automatic index on mangas_categories(manga_id) 
12-30 12:52:23.209 31017 31017 E BufferQueueProducer: Unable to open libpenguin.so: dlopen failed: library "libpenguin.so" not found.
